test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/services/chat-states/ChatStates.spec.js

```diff
@@ -397,6 +511,26 @@ describe("Chat States", () => {
             expect(container.transcript.addSystemMsg).toHaveBeenCalledWith({msg: "Agent 'JoeBloggs' loses connection"});
         });
 
+        it("reports agent joins conference", () => {
```

Comment:
This test should add something (not sure what?!) to the document with the .agent-joins-conference id before the handle message, and assert it does not remain on the document after the call. Could be a tricky one!